cmd/vpu_plugin/README.md

```diff
@@ -1,75 +1,240 @@
-# Build and test Intel VPU device plugin for Kubernetes
+# Intel VPU device plugin for Kubernetes
 
-### Get source code:
-```
-$ mkdir -p $GOPATH/src/github.com/intel/
-$ cd $GOPATH/src/github.com/intel/
-$ git clone https://github.com/intel/intel-device-plugins-for-kubernetes.git
-```
-Note: to get VCAC-A card running hddl, please refer to:
+# Table of Contents
+
+* [Introduction](#introduction)
+* [Installation](#installation)
+    * [Getting the source code](#getting-the-source-code)
+    * [Verify node kubelet config](#verify-node-kubelet-config)
+    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
+        * [Build the plugin image](#build-the-plugin-image)
+        * [Deploy plugin DaemonSet](#deploy-plugin-daemonset)
+    * [Deploy by hand](#deploy-by-hand)
+        * [Build the plugin](#build-the-plugin)
+        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
+    * [Verify plugin registration](#verify-plugin-registration)
+    * [Testing the plugin](#testing-the-plugin)
+
+# Introduction
+
+The VPU device plugin supports Intel VCAC-A card:
+(https://www.intel.com/content/dam/www/public/us/en/documents/datasheets/media-analytics-vcac-a-accelerator-card-by-celestica-datasheet.pdf).
```

Comment:
there are other cards too, right? perhaps list all of them?

I'd prefer we create links `[Intel VCAC-A](url)`.

Reply:
done